src/Chat/utils/commands.ts

```diff
@@ -5,6 +5,8 @@ import { ChatCommands, KnownNoticeMessageIds, Commands } from '../../twitch'
 
 import * as utils from '../../utils'
 
+const EVENT_TIMEOUT_LIMIT = 10000
```

Comment:
Please move to and import from `src/Chat/constants.ts`

Reply:
Sorry, forget which directory we are in; please ignore.

Reply:
I used the existing `COMMAND_TIMEOUT` constant. If you think I should really create a new one, please, reopen this conversation. :)